coalib/output/ConsoleInteraction.py

```diff
@@ -967,7 +970,8 @@ def print_bears(bears,
             show_bear(bear,
                       show_description,
                       show_params,
-                      console_printer)
+                      console_printer,
+                      args)
 
 
```

Comment:
Remove one line . 2 is an overkill

Reply:
1. see above
2. PEP8 defines that functions on module level shall have 2 empty lines in between.